index.php

```diff
@@ -11,19 +10,25 @@
 			Typekit.load();
 		} catch ( e ) {
 		}</script>
-	<script src="https://maps.googleapis.com/maps/api/js"></script>
-	<?php if ( defined( 'HM_DEV' ) ) : ?>
+	<script src="https://maps.googleapis.com/maps/api/js&key=<?php if ( defined( 'HM_GOOGLE_API_KEY' ) ) echo HM_GOOGLE_API_KEY ?>"></script>
```

Comment:
I dont think so. its a HM project and multimatley. there is HM_DEV in places too.